Update README.md

markkdamm opened the pull request:
I added information about security in the readme file as some users have reported getting their api keys stolen

Commit:
Update README.md

I added information about security in the readme file as some users have reported getting their api keys stolen

surjithctly commented:
Thanks for the PR.

Actually, the access key can be public. Its not a secret key. However I can see i mentioned `apikey` somewhere which confuses users. Can you update it to `access_key`? Also in security, mention that it can be public, its an alias to your public email. If they want domain lock, can purchase Pro.

Commit:
Update README.md

I updated the security details and I replaced 'API_key' with accesskey to prevent further confusion

markkdamm commented:
Done!

surjithctly commented:
Thank you! 🫡
